back-end/src/main/java/kr/co/ssalon/dto/TicketDTO.java

```diff
@@ -0,0 +1,15 @@
+package kr.co.ssalon.dto;
+
+import kr.co.ssalon.domain.entity.*;
+import lombok.*;
+
+@Getter
+@Setter
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class TicketDTO {
+    private Long id;
+    private Meeting meeting;
```

Comment:
저도 판단이 잘 안 서는데, 아래처럼 id값만 들고 있는 게 맞지 않을까요?
```suggestion
    private Long meetingId;
```

Reply:
id 만 들고 있는 걸로 바꾸겠습니다.